sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -2618,8 +2618,9 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
    */
   override def visitDescribeNamespace(ctx: DescribeNamespaceContext): LogicalPlan =
     withOrigin(ctx) {
+      val nameParts = visitMultipartIdentifier(ctx.multipartIdentifier())
       DescribeNamespaceStatement(
-        visitMultipartIdentifier(ctx.multipartIdentifier()),
+        UnresolvedNamespace(nameParts),
```

Comment:
let's not do this change now. We should only use `UnresolveNamespace` and `UnresolvedTable` in the COMMENT ON command, and migrate other commands later in separated PRs.

Reply:
OK